CHANGELOG.md

```diff
@@ -76,6 +76,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Martin Fiebig](https://github.com/mfiebig)
   [#8798](https://github.com/CocoaPods/CocoaPods/pull/8798)
 
+* Always suggest `pod repo update` on dependency resolution conflict,
```

Comment:
This needs to move to master section

Reply:
Done. Bad automatic merge. How do you guys deal with this, merge manually all the time?